doc/source/tutorials/go_models.rst

```diff
@@ -0,0 +1,85 @@
+to the `web server <http://pomalab.ippt.pan.pl/GoContactMap/>`_.
+
+The contact map is then used in your martinize2 command:
+
+``martinize2 -f protein.pdb -o topol.top -x cg_protein.pdb -ff martini3001 -dssp -go contact_map.out``
+
+
+Without any further additions, this will:
+ 1) Generate virtual sites with the atomname ``CA`` directly on top of all the backbone beads in your protein.
+    Each ``CA`` atom has an underlying atomtype (see below), which has a default name specified using the
+    ``-go-moltype`` flag.
+ 2) Use the contact map to generate a set of non-bonded parameters between specific pairs of ``CA`` atoms in your molecule
+    with strength 9.414 kJ/mol (changed through the ``-go-eps`` flag).
+ 3) Eliminate any parameters which are shorter than 0.3 nm and longer than 1.1 nm, or are closer than 3 residues in the
+    molecular graph. These options are flexible through the ``-go-low`` and ``-go-up`` flags.
+ 4) If the contact map finds any atoms within contact range defined, but are *also* within 3 residues of each other,
+    then the contacts are removed. This is defined through the ``-go-res-dist`` flag.
+
+As a result, along with the standard output of martinize2 (ie. itp files for your molecules, a generic .top file,
+a coarse grained structure file), you will get two extra files: ``go_atomtypes.itp`` and ``go_nbparams.itp``. The atomtypes
+file defines the new virtual sites as atoms for your system, and the nbparams file defines specific non-bonded
+interactions between them.
+
+For example, ``go_atomtypes.itp`` looks like any other ``[ atomtypes ]`` directive::
+
+ [ atomtypes ]
+ molecule_0_1 0.0 0 A 0.00000000 0.00000000
+ molecule_0_2 0.0 0 A 0.00000000 0.00000000
+ molecule_0_3 0.0 0 A 0.00000000 0.00000000
+ molecule_0_4 0.0 0 A 0.00000000 0.00000000
+ molecule_0_5 0.0 0 A 0.00000000 0.00000000
+ ...
+
+Similarly, ``go_nbparams.itp`` looks like any ``[ nonbond_params ]`` directive (obviously, the exact parameters here
+depend on your protein)::
+
+ [ nonbond_params ]
+ molecule_0_17 molecule_0_13 1 0.59354169 9.41400000 ;go bond 0.666228018941817
+ molecule_0_18 molecule_0_14 1 0.53798937 9.41400000 ;go bond 0.6038726468003999
+ molecule_0_19 molecule_0_15 1 0.51270658 9.41400000 ;go bond 0.5754936778307316
+ molecule_0_22 molecule_0_15 1 0.73815666 9.41400000 ;go bond 0.8285528398039018
+ molecule_0_22 molecule_0_18 1 0.54218134 9.41400000 ;go bond 0.6085779754055839
+ molecule_0_23 molecule_0_19 1 0.53307395 9.41400000 ;go bond 0.5983552758317587
+ ...
+
+To activate your Go model, you can simply include these files in your master ``martini_v3.0.0.itp`` file with the
+following commands::
+
+ sed -i "s/\[ nonbond_params \]/\#ifdef GO_VIRT\n\#include \"go_atomtypes.itp\"\n\#endif\n\n\[ nonbond_params \]/" martini_v3.0.0.itp
+
+ echo -e "\n#ifdef GO_VIRT \n#include \"go_nbparams.itp\"\n#endif" >> martini_v3.0.0.itp
```

Comment:
I'm not the biggest fan of this. I think a short code block with what to add to the itp or top file is clearer